README.md

```diff
@@ -8,69 +8,134 @@
   <em>CLI to work with your AsyncAPI files. Currently supports validation, but it is under development for more features.</em>
 </p>
 
-### Install
-```bash
-$ npm install --global @asyncapi/cli
+<p align="center">
+<a href="https://github.com/asyncapi/cli/blob/master/LICENSE"><img alt="GitHub license" src="https://img.shields.io/github/license/asyncapi/cli"></a>
+<a href="https://github.com/asyncapi/cli/actions/workflows/if-nodejs-pr-testing.yml">
+<img src="https://github.com/asyncapi/cli/actions/workflows/if-nodejs-pr-testing.yml/badge.svg" alt="PR testing - if Node project"  />
+</a>
+<a href="https://www.npmjs.com/package/@asyncapi/cli">
+<img alt="npm" src="https://img.shields.io/npm/dw/@asyncapi/cli">
+</a>
+
+</p>
+
+## Table of contents
+
+<!-- toc -->
+
+- [Requirements](#requirements)
+  * [Installation](#installation)
+- [Usage](#usage)
+- [API Reference](#api-reference)
+  * [`validate`](#validate)
+  * [`context`](#context)
+- [Contributing](#contributing)
+  * [Set up development enviornment](#set-up-development-enviornment)
+
+<!-- tocstop -->
+
+## Requirements
+To run `@asyncapi/cli`, you'll need Node.js >=v10
+
+Run this terminal command to check your Node.js version:
```

Comment:
Maybe it was here, or maybe it was in another repo, but did I give you a hint to describe the requirements in table form, such as?

- NodeJS >= v10 

Was this removed during the review from Łukasz?

Overall good, I have only this question.

Reply:
Maybe I missed it I will update it later, 👍🏻